### DISCUSSION TITLE
Duplicated notifications in mobile app

### DISCUSSION THREAD

**Discussion by daneah** [community/community#21323] [Mobile]
The more recent versions of the mobile app have stopped tracking the status of notifications properly. Items that I’ve long since reviewed and marked as done on the web continue to show up in the notifications on mobile. They can be cleared properly there, but the duplication between web and mobile has become bothersome.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.74.0 (74121991)
Device: iPhone 7 Plus (iOS 15.5)
</details>

**Comment by aconsuegra** [comment]
Hi @daneah, thanks for getitng in touch and for the bug report. I'm having some problems to reproduce the issue that you describe, do you have any filtering applied to the notifications in the iOS app?

**Reply by daneah** [reply]
@aconsuegra I don't. I'm able to reproduce this actively right now—I have 3 unread and 0 read notifications on web, but I have 3 unread and 3 read notifications on mobile.

**Reply by aconsuegra** [reply]
Interesting... Is it possible that the filter to display only unread notifications is enabled in web but not in the mobile app?

**Reply by daneah** [reply]
@aconsuegra wow, just checked and indeed my web client has been set to "Unread" basically since all time—this was hard for me to even find 😆 I think we're all good here.

<img width="150" alt="Screen Shot 2022-07-27 at 07 55 19" src="https://user-images.githubusercontent.com/1808306/181240759-5f0f71aa-95d4-4b5b-8d24-8deccc42632b.png">

**Comment by jfullstackdev** [comment]
this might be related too, I still have this issue by now

![image](https://user-images.githubusercontent.com/47092464/181179123-8d0263e6-d56f-47e7-bbd1-4c82049b8498.png)

although it's a known bug in the past at the standard browser
